pkg/thrap/deployments.go

```diff
@@ -33,15 +33,15 @@ type Deployments struct {
 }
 
 // NewDeployments returns a new Deployments instance for a given project
-func NewDeployments(t *Thrap, proj pb.Project) *Deployments {
+func NewDeployments(t *Thrap, proj pb.Project, version string) *Deployments {
```

Comment:
NewDeployments manages all deploys for a project.  Version should not be in the constructor.